### DISCUSSION TITLE
What happens to the data on the node when the cluster node leaving?

### DISCUSSION THREAD

**Discussion by lconcteau** [emqx/emqx#3936] [Q&A] [answered]
I typed the command:./bin/emqx_ctl cluster leave  when the cluster is running, but what happens to the data on this node? drop or 
move to the another node?

**Comment by zmstone** [comment, marked as answer]
Hi. Sorry for the late reply.

So far, EMQ X clustered nodes all have the full replica of the data.
It should be safe to delete the data for the nodes which have already left the cluster.
